src/main/java/org/geonetwork/domain/Group.java

```diff
@@ -68,16 +68,19 @@ public class Group {
   @JoinColumn(name = "defaultcategory_id")
   private Category defaultcategory;
 
+  @Builder.Default
   @ManyToMany
   @JoinTable(
       name = "group_category",
       joinColumns = @JoinColumn(name = "group_id"),
       inverseJoinColumns = @JoinColumn(name = "category_id"))
   private Set<Category> categories = new LinkedHashSet<>();
 
+  @Builder.Default
   @OneToMany(mappedBy = "iddes")
   private Set<Groupsde> groupsdes = new LinkedHashSet<>();
```

Comment:
@fxprunayre if I understand correctly each time you supply a value like this we need to hint the builder that this should be used as a default value.

Reply:
Not really sure, the domain was created using Intellij JPA wizard.